scripts/setup_relative_calculation.py

```diff
@@ -0,0 +1,151 @@
+import openeye.oechem as oechem
+import rdkit.Chem as chem
+import yaml
+from perses.rjmc import topology_proposal, geometry
+from perses.annihilation import relative
+from perses.tests import utils
+import simtk.unit as unit
+from openmmtools.constants import kB
+
+
+
+def load_sd_file_rdkit(mol_filename):
+    mol_file = open(mol_filename, 'r')
+    suppl = chem.SDMolSupplier(mol_file)
+    mols = [mol for mol in suppl]
+    return mols[0]
+
+def load_mol_file(mol_filename):
+    """
+    Utility function to load
+    Parameters
+    ----------
+    mol_filename : str
+        The name of the molecule file. Must be supported by openeye.
+
+    Returns
+    -------
+    mol : oechem.OEMol
+        the molecule as an OEMol
+    """
+    ifs = oechem.oemolistream()
+    ifs.open(mol_filename)
+    #get the list of molecules
+    mol_list = [mol for mol in ifs.GetOEMols()]
+    #we'll always take the first for now
+    return mol_list[0]
+
+
+
+class RelativeFreeEnergySetup(object):
```

Comment:
It's not finished. I think I'm going to finish it in another PR. I can remove it for now if that's better.